### DISCUSSION TITLE
Make tokenvault free

### DISCUSSION THREAD

**Discussion by ghost** [standardnotes/forum#2735] [Suggestions]
Standard Notes is a privacy focused app. So some security extensions like tokenvault can be provided for free. I agree the developers want to get money too but at least some extensions can be made free of cost.

Thank You.

**Comment by dandv** [comment]
Can you please edit the title of your issue to be more descriptive?

**Comment by JaspalSuri** [comment]
Hi @A12341A,

Please see [this support article](https://standardnotes.org/help/24/why-is-two-factor-authentication-a-paid-feature) on why certain features are paid features. While it answers a different question, the answer is largely the same. The Extended plans helps support our capacity to maintain the extensions that we offer.
